test/fixtures/postgresql-ha/main.tf

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-provider "google" {}
-
+provider "google" {
```

Comment:
think this sould be moved to examples as well

Reply:
This module does not use examples for running tests, So we have to declare providers in fixtures.